rollup.config.umd.js

```diff
@@ -5,10 +5,11 @@ export default {
   input: 'dist/index.js',
   name: 'ng.instantsearch',
   sourcemap: true,
-  externals: [
+  external: [
     '@angular/core',
     '@angular/common',
-    'instantsearch.js',
+    'instantsearch.js/es',
```

Comment:
if this is external, does it mean you still need to import it separately?

Reply:
Nope, it just means it is not bundled into the final file. But since it is a dependency it will be installed with npm as well.